src/dune_rules/string_expander.mli

```diff
@@ -0,0 +1,5 @@
+(** String expansion in the action_builder monad *)
+
+open Import
+
+include String_with_vars.Expander with type 'a app := 'a Action_builder.t
```

Comment:
IMO, this functor application should be in `Action_builder`. E.g. `Action_builder.Expander`. It's an overkill to have a separate file for a functor application.

Reply:
Seems reasonable, I did the change.